index.html

```diff
@@ -134,27 +134,44 @@ <h2>
         <code><dfn>note_taking</dfn></code> member
       </h2>
       <p>
-        The <code>note_taking</code> member of the <a data-cite=
+        The `note_taking` member of the <a data-cite=
         "appmanifest#web-application-manifest">Web Application Manifest</a> is
-        an <a data-cite="appmanifest#dfn-object">object</a> that contains
-        information related to note-taking. It has the following members:
+        an [=ordered map=] that contains information related to note-taking. It
+        has the following members:
       </p>
       <ul>
         <li>[=note_taking/new_note_url=]
         </li>
       </ul>
+      <p>
+        A user agent MAY use these members to treat the web application
+        differently as an application with note-taking capabilities (e.g.,
+        integrate with operating system note-taking surfaces).
+      </p>
       <section>
         <h3>
           <code><dfn data-dfn-for="note_taking">new_note_url</dfn></code> member
         </h3>
         <p>
-          The [=note_taking=] <code>new_note_url</code> member is a <a>URL</a>
-          [=manifest/within scope=] of the manifest, to open when the user
-          wants to take a new note using the web application (e.g., from an
-          operating system shortcut icon or keyboard shortcut). Defining this
-          member indicates that the web application is a note-taking
-          application.
+          The [=note_taking=] `new_note_url` member is a [=string=] that
+          represents the <a data-cite="url#concept-url">URL</a> the developer
+          would prefer the user agent load when the user wants to take a new
+          note using the web application (e.g., from an operating system
+          shortcut icon or keyboard shortcut).
+        </p>
+        <p>
+          The `new_note_url` member is purely advisory,
+          and a user agent MAY <a data-cite="appmanifest#dfn-ignore">ignore</a>
+          it or provide the end-user the choice to modify it or to not use it.
```

Comment:
Sentence is hard to parse (mainly because of the nested "or"s).

(i.e. it could be read either as:

> (ignore it) or (provide the end-user the choice to modify it) or (to not use it)

or, as intended:

> (ignore it) or (provide the end-user the choice to (modify it) or (to not use it))

Perhaps reword as

"... MAY ignore it entirely, or provide the end-user the choice of whether to use it. The user agent MAY provide the end-user the choice to modify it."

Reply:
> "object" (pre-processing)
Per https://github.com/w3c/manifest/issues/984 I am kinda assuming we will have an infra object, not a JSON one. But that hasn't landed so I guess I will continue the charade where JSON objects are a thing that you can do operations on :stuck_out_tongue: 

Done.